ts/src/renderer/phaser/classes/PhaserAttributeBar.ts

```diff
@@ -79,13 +78,15 @@ class PhaserAttributeBar extends Phaser.GameObjects.Container {
 		bar.fillStyle(Phaser.Display.Color
 			.HexStringToColor(data.color)
 			.color);
-		bar.fillRoundedRect(
-			-w / 2,
-			-h / 2,
-			w * data.value / data.max,
-			h,
-			borderRadius
-		);
+		if (data.value !== 0) {
+			bar.fillRoundedRect(
+				-w / 2,
+				-h / 2,
+				Math.max(w * data.value / data.max, 10),
```

Comment:
Instead of arbitrary value 10 we should use something based on `borderRadius` value